intezer_sdk/consts.py

```diff
@@ -104,3 +104,5 @@ class OnPremiseVersion(enum.IntEnum):
 USER_AGENT = f'intezer-python-sdk-{__version__}'
 CHECK_STATUS_INTERVAL = 1
 SCAN_TYPE_OFFLINE_ENDPOINT_SCAN = 'offline_endpoint_scan'
+SCAN_MAX_WORKERS = 4
```

Comment:
Make sure you can configure this via the CLI without changing this const in runtime